src/services/formatting/smartIndenter.ts

```diff
@@ -279,7 +279,14 @@ module ts.formatting {
 
         function getActualIndentationForListItem(node: Node, sourceFile: SourceFile, options: EditorOptions): number {
             let containingList = getContainingList(node, sourceFile);
-            return containingList ? getActualIndentationFromList(containingList) : Value.Unknown;
+
+            if (containingList) {

```

Comment:
`getIndentationForNodeWorker` in that situation receives CallExpression node and the indentation for this node also affects:

```
Promise.
    .resolve()/**/
    .then();
```

I'll investigate this more.